Source/WebCore/rendering/style/StyleRareInheritedData.cpp

```diff
@@ -69,6 +69,8 @@ struct GreaterThanOrSameSizeAsStyleRareInheritedData : public RefCounted<Greater
 #if ENABLE(DARK_MODE_CSS)
     StyleColorScheme colorScheme;
 #endif
+    TextSpacingTrim textSpacingTrim;
+    TextAutospace textAutospace;
```

Comment:
cc @heycam, who has been looking at moving things into / out of `StyleRareInheritedData`

Reply:
I think this is appropriate in RareInheritedData, I don't expect super wide usage of these properties.